lavamoat/build-system/policy.json

```diff
@@ -2429,6 +2450,45 @@
         "process.version": true
       }
     },
+    "fsevents": {
```

Comment:
@kumavis @mcmire do we have a good way to evaluate changes to the lavamoat policy file? Or should we assume they're safu?

Reply:
I think as long as you can say `yarn start` and as long as CI works (since it runs `yarn build prod`) then you can assume these changes are safe? 🤔